### PR TITLE
Club Pages (all clubs, club pages)

### PR DESCRIPTION
**Pull request by Jchuang7**
Added dynamic routing to each club page, test functionalities since there's no data to test yet.

### CODE REVIEW

File: client/src/App.tsx [hunk 1]
```diff
@@ -9,3 +9,32 @@ ReactDOM.createRoot(document.getElementById("root")!).render(
     <Router />
   </React.StrictMode>
 );
+
+interface Listitem {
+  text: string;
+}
+
+function createUL(listItems: Listitem[]): void {
```
Comment (varangill) [inline client/src/App.tsx]: React has custom ways to create a component for objects within an array by calling `map` on the array that we should use, it's a lot cleaner and avoids raw JS DOM manipulation. @kravich2 's [recent PR here](https://github.com/varangill/ClubHub/pull/9/files#diff-26cee6f83578cbe23a73e0f3c317f0764a2a250bc18cc362c69436f42c804730R67) has an example of it

File: client/src/Club.css [hunk 2]
```diff
@@ -0,0 +1,28 @@
+
+  .custom-header {
+    background-color: #333;
+    color: white;
+    padding: 10px; /* Add padding to the entire header */
```
Comment (varangill) [inline client/src/Club.css]: Not a blocker but we don't really need comments on CSS properties (unless it's something really weird/confusing), you don't have to remove them here, just a head's up for the future to save some time

File: client/src/App.tsx (hunk 1 above)
Comment (varangill) [reply client/src/App.tsx]: Also this sort of page-specific logic shouldn't be in `App.tsx`

**Review by varangill — CHANGES_REQUESTED** [review]
Comment on the createUL function, let us know if you need help

Also could you post an image in the PR itself for each page that's created/updated in the PR?

**Comment by Jchuang7** [comment]
image
Screen Shot 2024-01-09 at 1 49 07 PM
image
image

I didn't set up example data yet so the information for members and announcments are the same for now.

**Pull request closed without merging**